test/testdata/deviations/keyword_method_names.rb

```diff
@@ -73,14 +73,6 @@ def method_named_do(x)
     do # error: unexpected token
 end
 
-def method_named_else(x)
```

Comment:
why was this deleted?

Reply:
> 1. `useless begin; else; end` is now a syntax error (previously it was a warning)

because it produces a syntax error that is completely unrelated to this test

Reply:
does it? It checks that we can call a method called `else`.
Can we keep it?

Reply:
I'm not 100% sure, but I thought this test asserts that the code like
```
a.
  <keyword>
```
is interpreted as `a.<keyword>` (and not as `a.<keyword as a method name>`) to speedup language server.

It works with `else`, but `begin; else; end` is a syntax error now, and so
```
a.
  else
```
becomes `tIDENTIFIER(A) -> tDOT -> tELSE` which gives a different error now.

I can update an error message, but I don't know if the language server still works for this case. @jez Could you take a look please?

Reply:
@DarkDimius I've just added back this test. Here's the original method:
``` ruby
def method_named_else(x)
  x.else # ok
  x
    .else # ok
  x.
    else # error: unexpected token
end
```

The last part now produces an error, and so the whole method body gets reduced into an `error` (and it's completely useless now). I added back first two statements. Also, the CI seems to failing now. I didn't change anything except input/output files, but it fails in the completely unrelated job.

Reply:
>  I added back first two statements. Also, the CI seems to failing now.

you'll need to rebase, we did some changes to the compilation toolchain for the weekend that aren't correctly invalidated by the build cache.

Reply:
@DarkDimius done